assets/js/blocks/breadcrumbs/index.tsx

```diff
@@ -10,6 +11,23 @@ import { Icon, queryPagination } from '@wordpress/icons';
 import metadata from './block.json';
 import edit from './edit';
 
+const featurePluginSupport = {
+	...metadata.supports,
+	...( isFeaturePluginBuild() && {
+		typography: {
+			fontSize: true,
+			lineHeight: true,
```

Comment:
What do you think about replacing this with:
```suggestion
			...metadata.supports.typography,
```

This way, if we make any changes in `block.json`, they will be reflected here.

Reply:
Good call, updated!